pkg/ippoolmanager/ippool_manager.go

```diff
@@ -195,6 +196,37 @@ func (im *ipPoolManager) genRandomIP(ctx context.Context, ipPool *spiderpoolv2be
 	}
 	resIP := availableIPs[0]
 
+	assignIpMap := make(map[string][]string)
+	if anno, ok := pod.Annotations["ipam/ips"]; ok {
```

Comment:
1 it likely does not follow the annotation prefix of the spiderpool 

2 does it likely make ip conflict with ip assigned in this way of spiderippool